helm-charts/konk/scripts/provision.sh

```diff
@@ -29,3 +29,10 @@ then
     --from-file=/etc/kubernetes/admin.conf
   kubectl -n $NAMESPACE label secret $FULLNAME-kubeconfig $LABELS
 fi
+
+sleep 1m
```

Comment:
Please replace this with a watch/wait. Something like this but on the deployment resource will probably work:
https://github.com/infobloxopen/konk/blob/e03343b284655e575ad1a91b95c927ef03ba7b70/.github/workflows/pr.yaml#L25

Reply:
Replaced `sleep` with `kubectl wait`.